components/ide/jetbrains/backend-plugin/src/main/resources/META-INF/plugin.xml

```diff
@@ -22,18 +22,114 @@
-        <applicationService serviceImplementation="io.gitpod.jetbrains.remote.services.HeartbeatService" preload="true"/>
+        <applicationService serviceImplementation="io.gitpod.jetbrains.remote.services.HeartbeatService"
+                            preload="true"/>
         <applicationService serviceImplementation="io.gitpod.jetbrains.remote.GitpodManager" preload="true"/>
         <applicationService serviceImplementation="io.gitpod.jetbrains.remote.GitpodPortsService" preload="true"/>
-        <notificationGroup id="Gitpod Notifications" displayType="BALLOON" isLogByDefault="false" />
+        <notificationGroup id="Gitpod Notifications" displayType="BALLOON" isLogByDefault="false"/>
         <httpRequestHandler implementation="io.gitpod.jetbrains.remote.GitpodCLIService"/>
-        <projectService serviceImplementation="io.gitpod.jetbrains.remote.GitpodClientProjectSessionTracker" client="guest" preload="true"/>
+        <projectService serviceImplementation="io.gitpod.jetbrains.remote.GitpodClientProjectSessionTracker"
+                        client="guest" preload="true"/>
         <projectService serviceImplementation="io.gitpod.jetbrains.remote.GitpodProjectManager" preload="true"/>
-        <projectService serviceImplementation="io.gitpod.jetbrains.remote.GitpodTerminalService" client="guest" preload="true"/>
-        <gateway.customization.name implementation="io.gitpod.jetbrains.remote.GitpodGatewayClientCustomizationProvider"/>
-        <gateway.customization.performance id="gitpodMetricsControl" order="before cpuControl" implementation="io.gitpod.jetbrains.remote.GitpodMetricControlProvider"/>
-        <gateway.customization.metrics id="gitpodMetricsProvider" implementation="io.gitpod.jetbrains.remote.GitpodMetricProvider" />
-        <registryKey key="gitpod.autoJdk.disabled" defaultValue="false" description="Disable auto-detection of JDK for the project and its modules" restartRequired="true"/>
+        <projectService serviceImplementation="io.gitpod.jetbrains.remote.GitpodTerminalService" client="guest"
+                        preload="true"/>
+        <gateway.customization.name
+                implementation="io.gitpod.jetbrains.remote.GitpodGatewayClientCustomizationProvider"/>
+        <gateway.customization.performance id="gitpodMetricsControl" order="before cpuControl"
+                                           implementation="io.gitpod.jetbrains.remote.GitpodMetricControlProvider"/>
+        <gateway.customization.metrics id="gitpodMetricsProvider"
+                                       implementation="io.gitpod.jetbrains.remote.GitpodMetricProvider"/>
+        <registryKey key="gitpod.autoJdk.disabled" defaultValue="false"
+                     description="Disable auto-detection of JDK for the project and its modules"
+                     restartRequired="true"/>
     </extensions>
 
+    <actions>
+        <action id="io.gitpod.jetbrains.remote.actions.DashboardAction"
```

Comment:
Is there any specific logic around the order of the actions?

In VS Code all commands in the palette are sorted alphabetically, and it seems here we have a bit more flexibility to decide. 

I personally would either  go with alphabetical order or alternatively, with relevance. For example, 

- <Contextual to the workflow/workspace> (e.g. stop, go to dashboard)
- <Gitpod Generic> (e.g. docs, upgrade plan)
- <Social> (e.g. chat, twitter)

My preference is having relevant order, but I am interested to know other points of view. cc @felladrin @loujaybee

Reply:
I agree with ordering it by relevance. Currently it's ordered by what I thought was more relevant.
Let me know in which order do you think they should be?

Reply:
I think the order you gave is quite good actually, I got confused by the screenshot in which you search for actions and _Follow us on Twitter_ is the second option 😄

Reply:
That's a good point! It means it deserves a comment clarifying it on the PR description. I'll comment there, thanks!